java-frontend/src/test/java/org/sonar/java/se/MethodYieldTest.java

```diff
@@ -245,8 +250,19 @@ public void catch_class_cast_exception() throws Exception {
     assertThat(behaviorCache.values()).hasSize(1);
     MethodBehavior methodBehavior = behaviorCache.values().iterator().next();
     assertThat(methodBehavior.yields()).hasSize(2);
-    assertThat(methodBehavior.yields().get(1).resultIndex).isEqualTo(0);
-    assertThat(methodBehavior.yields().get(0).resultConstraint.isNull()).isTrue();
+    MethodYield[] expected = new MethodYield[] {
+      buildMethodYield(0, null, (Constraint) null),
```

Comment:
As far as i can tell you only use this method with a varargs being a single null constraint. So remove it from the parameter and hardcode the array in `buildMethodYield`.